docs/ibc/apps.md

```diff
@@ -238,10 +238,21 @@ DecodePacketData(encoded []byte) (CustomPacketData) {
 Then a module must encode its packet data before sending it through IBC.
```

Comment:
I realized that this file is actually not used in the docs website, so could probably be removed...

Reply:
I see, looks like we took this info and split it up into separate files? Lets open an issue whose task is to see if any info in here should be moved before deleting the file

Reply:
#2480